<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Figures - Mirnov Coil Data Acqusition</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="GraphicsLayoutWidget" name="figLayout" native="true">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_4"/>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="topwidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="8">
        <widget class="QCheckBox" name="filterCheckbox">
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;f&lt;span style=&quot; vertical-align:sub;&quot;&gt;min&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="lastShotButton">
         <property name="text">
          <string>Last shot</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Shot:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="15">
        <widget class="QLabel" name="signalsLabel">
         <property name="text">
          <string>Signals: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="12">
        <widget class="QLineEdit" name="filterFMax">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="10">
        <widget class="QLineEdit" name="filterFMin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QCheckBox" name="downsampleBox">
         <property name="text">
          <string>Downsample:</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="14">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="shotNumberInput">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="loadDataButton">
         <property name="text">
          <string>Load data</string>
         </property>
        </widget>
       </item>
       <item row="0" column="16">
        <widget class="QComboBox" name="signalArraySelector">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="18">
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLineEdit" name="downsampleFactorBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>22</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>20</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Factor</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="11">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;f&lt;span style=&quot; vertical-align:sub;&quot;&gt;max&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="13">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;[kHz]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="bottomwidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="6">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>t&lt;sub&gt;1&lt;/sub&gt;:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="lowerTLim">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>1050</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Lower limit [ms]</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="integrateDataButton">
         <property name="text">
          <string>Integrate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>t&lt;sub&gt;0&lt;/sub&gt;:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="15">
        <widget class="QPushButton" name="seeAloneButton">
         <property name="text">
          <string>See alone</string>
         </property>
        </widget>
       </item>
       <item row="1" column="12">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="7">
        <widget class="QLineEdit" name="upperTLim">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>1250</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Upper limit [ms]</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="fftButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>FFT</string>
         </property>
        </widget>
       </item>
       <item row="1" column="14">
        <widget class="QComboBox" name="coilDataRetrievalSelector"/>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="spectrogramsButton">
         <property name="text">
          <string>Spectrograms</string>
         </property>
        </widget>
       </item>
       <item row="1" column="16">
        <widget class="QPushButton" name="singleSpectrogramButton">
         <property name="text">
          <string>Spectrogram</string>
         </property>
        </widget>
       </item>
       <item row="1" column="9">
        <widget class="QLineEdit" name="spgramNperseg">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>512</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>nperseg</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>nperseg:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="11">
        <widget class="QLineEdit" name="spgramNoverlap">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>500</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>noverlap</string>
         </property>
        </widget>
       </item>
       <item row="1" column="10">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>noverlap:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="13">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Signal:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionCheck_DAQ"/>
    <addaction name="actionSave_figure"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_figure">
   <property name="text">
    <string>Save figure</string>
   </property>
  </action>
  <action name="actionCheck_DAQ">
   <property name="text">
    <string>Check DAQ</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
